<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageevenement</class>
 <widget class="QDialog" name="pageevenement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-50</y>
     <width>1451</width>
     <height>811</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img/R.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="LCD">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>1211</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* styles.qss */

* {
    font-family: &quot;Playfair Display&quot;, serif; /* Fallback to generic serif font if Playfair Display is not available */
    color: black;
}

/* Style for QGroupBox */
QGroupBox {
    border: 2px solid #e67e22; /* Change to a bright orange color */
    border-radius: 5px;
    padding: 10px;
    color: white; /* Set text color inside group box */
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
}

/* Style for QLineEdit (input box) */
QLineEdit {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

/* Style for QComboBox */
QComboBox {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    color: black;
}

/* Style for QDateEdit */
QDateEdit {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px;
    color: white;
}

/* Style for QLabel */
QLabel {
    color: #e67e22; /* Change to a bright orange color */
    font-size: 16px;
    font-weight: bold;
    margin: 5px;
}

/* Style for QPushButton */
QPushButton {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px 10px;
    color: white;
}

/* Additional styling for better appearance */
QPushButton:hover {
    background-color: #d35400; /* Darker orange color on hover */
}

QPushButton:pressed {
    background-color: #a04000; /* Even darker orange color on press */
}

/* Style for QTableView */
QTableView {
    background-color: rgba(255, 255, 255, 0.8); /* Slightly transparent white background */
    border: 5px solid #e67e22; /* Creative thick orange border */
    border-radius: 15px; /* Rounded corners for a more creative look */
    selection-background-color: #e67e22; /* Change to a bright orange color */
}

/* Style for QHeaderView */
QHeaderView::section {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    color: white;
    padding: 5px;
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
}

/* Style for QTableWidgetItem */
QTableView QTableWidget::item {
    background-color: rgba(255, 255, 255, 0.8); /* Slightly transparent white background */
    color: black;
    padding: 5px;
}

QTableView QTableWidget::item:selected {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    color: white;
}

QTableView QTableWidget::item:alternate {
    background-color: #ecf0f1;
    color: black;
}

/* Style for QTabWidget */
QTabWidget {
    background-color: #ffffff; /* Set background color for the tab widget */
}

/* Style for QTabBar */
QTabBar {
    background-color: #e67e22; /* Set the background color for the tab bar */
}

/* Style for selected tab */
QTabBar::tab:selected {
    background-color: #d35400; /* Darker orange color for the selected tab */
    color: white; /* Text color for the selected tab */
    border: 2px solid #a04000; /* Border style for the selected tab */
    border-bottom: none; /* Remove bottom border to create a top border effect */
}

/* Style for unselected tabs */
QTabBar::tab {
    background-color: #e67e22; /* Default background color for tabs */
    color: white; /* Default text color for tabs */
    border: 2px solid #d35400; /* Default border style for tabs */
    border-bottom: none; /* Remove bottom border to create a top border effect */
}

/* Style for unselected tabs on hover */
QTabBar::tab:hover {
    background-color: #d35400; /* Darker orange color on hover for unselected tabs */
}
</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QTableView" name="tab_events">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>581</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Titre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Date de debut</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Date de fin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre d'invitation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="titre_e">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nb_e">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>340</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>500</y>
       <width>102</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>500</y>
       <width>102</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>500</y>
       <width>102</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dated_e">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="datef_e">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom article</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_a">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Titre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sortAscButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>291</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>ASC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortDescButton">
     <property name="geometry">
      <rect>
       <x>752</x>
       <y>70</y>
       <width>291</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>DESC</string>
     </property>
    </widget>
    <widget class="QComboBox" name="location_e">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>400</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>location1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>location2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>location3</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>440</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Locations</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_chart">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>160</y>
       <width>641</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>440</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="refrech">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Refrech</string>
     </property>
    </widget>
    <zorder>timeEdit</zorder>
    <zorder>lab_chart</zorder>
    <zorder>tab_events</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>titre_e</zorder>
    <zorder>nb_e</zorder>
    <zorder>pb_ajouter</zorder>
    <zorder>pb_modifier</zorder>
    <zorder>pb_supprimer</zorder>
    <zorder>dated_e</zorder>
    <zorder>datef_e</zorder>
    <zorder>label_9</zorder>
    <zorder>nom_a</zorder>
    <zorder>radioButton</zorder>
    <zorder>label_7</zorder>
    <zorder>lineEdit</zorder>
    <zorder>sortAscButton</zorder>
    <zorder>sortDescButton</zorder>
    <zorder>location_e</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>refrech</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_56">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rcpt">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>501</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_53">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>511</width>
       <height>150</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="send">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>470</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_client">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>50</y>
       <width>341</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>ench</string>
    </attribute>
    <widget class="QLineEdit" name="prix_o">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>140</y>
       <width>251</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tab_oeuvre">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>80</y>
       <width>341</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="date_o">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>170</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="discription_o">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>270</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_o">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_a_2">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>320</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushB">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>140</y>
       <width>93</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LCD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushc1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Client 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushc2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Client 2</string>
     </property>
    </widget>
    <zorder>nom_o</zorder>
    <zorder>date_o</zorder>
    <zorder>discription_o</zorder>
    <zorder>nom_a_2</zorder>
    <zorder>prix_o</zorder>
    <zorder>tab_oeuvre</zorder>
    <zorder>pushB</zorder>
    <zorder>pushc1</zorder>
    <zorder>pushc2</zorder>
   </widget>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_24">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgba(230, 126, 34, 0.9); /* Slightly transparent orange background */
    border: 1px solid #d35400; /* Change to a darker orange color */
    border-radius: 5px;
    padding: 5px 10px;
    color: white;
}</string>
   </property>
   <property name="text">
    <string>retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
